src/components/Dashboard/StaffNotation.vue

```diff
@@ -164,6 +165,6 @@ export default class StaffNotation extends Vue {
 }
 
 ::v-deep .theme--light.v-list-item--disabled {
-  opacity: 0.38 !important;
+    opacity: 0.38 !important;
```

Comment:
This file seems to have 4 space tabs vs 2 space in most of the other files. 
Not a big deal but perhaps we convert this file to 2 space tabs in the `<style>`?

Reply:
This bugs you too? I'll fix it!

Reply:
Whatever you prefer sir! 😁

Reply:
Done and committed.